lib/application.js

```diff
@@ -667,6 +529,33 @@ Origin.prototype.ModulePreloader.defOpts = {
   }
 };
 
+function checkForUpdates(options, callback) {
+  // TODO shouldn't be checking for this here...
```

Comment:
Where would be a better place to check this? Could that be done as part of this PR?

I think we should avoid leaving TODOs in PRs where possible.